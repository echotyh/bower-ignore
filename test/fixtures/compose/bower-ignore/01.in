{
  "in": {
    "dependencies": {
      "bootstrap": "^3.3.6",
      "font-awesome": "fontawesome#^4.6.3"
    }
  },
  "force": false,
  "update": null
}
