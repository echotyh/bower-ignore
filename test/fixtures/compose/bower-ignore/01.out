{
  "bootstrap": {
    "ignore": [],
    "unignore": []
  },
  "font-awesome": {
    "ignore": [],
    "unignore": []
  }
}
